### DISCUSSION TITLE
Does this go-git library only support repositories hosted on github?

### DISCUSSION THREAD

**Discussion by charuag** [go-git/go-git#683] [Q&A]
Does this go-git library only support repositories hosted on github?

**Comment by pjbgf** [comment]
@charuag `go-git` supports any server that exposes the following protocols:

![image](https://user-images.githubusercontent.com/5452977/221988758-7c836e39-5702-4597-a0a6-2e447b59291c.png)


For latest support information, please refer to the [compatibility page](https://github.com/go-git/go-git/blob/master/COMPATIBILITY.md).
